js/ElementRendererProvider.js

```diff
@@ -76,6 +80,96 @@ export default class ElementRendererProvider {
       return divEl;
     });
 
+    this.set(TYPES.SUBMIT, (config): HTMLElement => {
+      const divEl = document.createElement('div');
+      divEl.className = 'lp-json-pollock-element-submit-button';
+
+      if (config.rtl) {
+        divEl.dir = 'rtl';
+        Utils.addClass(divEl, 'direction-rtl');
+      }
+
+      const btnEl = document.createElement('button');
+      btnEl.innerHTML = Utils.normalizeHtmlText(config.title);
+
+      if (config.tooltip) {
+        btnEl.title = config.tooltip;
+        btnEl.setAttribute('aria-label', config.tooltip);
+      }
+      if (config.style) {
+        const style = Utils.styleToCss(config.style);
+        const splitedStyle = Utils.extractFromStyles(style, 'background-color');
+        divEl.setAttribute('style', splitedStyle.extractedStyle);
+        btnEl.style.cssText = splitedStyle.style;
+      }
+
+      if (config.click && config.click.actions) {
+        btnEl.onclick = this.wrapAction(config.click);
+      }
+
+      divEl.appendChild(btnEl);
+
+      return divEl;
+    });
+
+    this.set(TYPES.CHECKBOX, (config): HTMLElement => {
+      const divEl = document.createElement('div');
+      divEl.className = 'lp-json-pollock-element-checkbox';
+
+      if (config.rtl) {
+        divEl.dir = 'rtl';
+        Utils.addClass(divEl, 'direction-rtl');
+      }
+
+      const labelEl = document.createElement('label');
+      const checkEl = document.createElement('input');
+      const tmp = config;
```

Comment:
is `tmp` needed?

Reply:
yes `tmp` is being used by `afterRender` function to attach the click listener

Reply:
you can use `config` on `afterRender`

Reply:
oh you are right. I will update that accordingly thank you!